src/meta/meta_backup_service.cpp

```diff
@@ -661,6 +661,20 @@ void policy_context::sync_backup_to_remote_storage_unlocked(const backup_info &b
 
 void policy_context::continue_current_backup_unlocked()
 {
+    if (_policy.is_disable) {
```

Comment:
policy is abandoned, so why need consider it?

Reply:
I added context in pr discription.